examples/cloud-backup-schedule/cloudBackupSchedule.json

```diff
@@ -15,37 +15,25 @@
       "Type": "String",
       "Default": ""
     },
-    "CloudProvider": {
-      "Type": "String",
-      "Default": "AWS"
-    },
-    "RegionName": {
-      "Type": "String",
-      "Default": "AP_NORTHEAST_2"
-    },
-    "ReplicationSpecId": {
-      "Type": "String",
-      "Default": ""
-    },
```

Comment:
Why are you modifying the example?

Reply:
because it was failing due to that structure that doesn't match the current spec, so I prefered to use the same example as in contract tests

Reply:
we should update the example with the correct structure instead of removing it so that customers know who to use it

Reply:
you're right, I reverted the example and tried again, it turns out the problem was I was not using a correct id format for that structure, it's well defined.